zulipterminal/ui_tools/views.py

```diff
@@ -1211,6 +1211,14 @@ def __init__(self, controller: Any, stream_id: int) -> None:
             label="Pinned to top", state=pinned_state, checked_symbol=CHECK_MARK
         )
         urwid.connect_signal(pinned_setting, "change", self.toggle_pinned_status)
+        desktop_notification = urwid.CheckBox(
+            label="Visual desktop notification",
```

Comment:
The "desktop" bit make end-users wonder whether this would only affect desktop notifications and not ZT's. Let's discuss it on the stream as well.